src/config/api.js

```diff
@@ -1,8 +1,8 @@
 import { create } from 'apisauce';
 
-const baseURL = 'http://wolox.com';
+const baseURL = process.env.REACT_APP_BASE_URL;
 
-if (baseURL === 'http://wolox.com') {
+if (baseURL === process.env.REACT_APP_BASE_URL) {
```

Comment:
This one is important. I don't think you were supposed to change this condition, the bootstrap wants to validate if the base url equals `'http://wolox.com'`. This is currently generating a `console.warn` in the development branch even despite the fact that the `baseURL` is properly initialized.